File: SIS104Menu/app/src/main/java/com/example/sis104menu/Ejercicios/EjercicioActivity.java
package com.example.sis104menu.Ejercicios;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import com.example.sis104menu.R;

public class EjercicioActivity extends AppCompatActivity {
    Button button_Ecuacion, button_Salir;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_ejercicio);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });


        button_Ecuacion = findViewById(R.id.button_Ecuacion2doGrado);
        button_Salir = findViewById(R.id.button_SalirEjercicio);


        button_Ecuacion.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(EjercicioActivity.this, Ecuacion2dogradoActivity.class);
                startActivity(intent);
            }
        });

        button_Salir.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                finish();
            }
        });
    }
}

File: SIS104_Kotlin/app/src/main/java/com/example/sis104_kotlin/ejercicios/EjercicioActivity.kt
package com.example.sis104_kotlin.ejercicios

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import com.example.sis104_kotlin.R

class EjercicioActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_ejercicio)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }

        val buttonEcuacion = findViewById<Button>(R.id.buttonEcuacion)
        val buttonComplejos = findViewById<Button>(R.id.buttonComplejos)
        val buttonEjercicioSalir = findViewById<Button>(R.id.buttonEjercicioSalir)

        buttonEcuacion.setOnClickListener {
            val intent = Intent(this@EjercicioActivity, EcuacionActivity::class.java)
            startActivity(intent)
        }

        buttonComplejos.setOnClickListener {
            val intent = Intent(this@EjercicioActivity, ComplejoActivity::class.java)
            startActivity(intent)
        }

        buttonEjercicioSalir.setOnClickListener {
            finish()
        }
    }
}